### PR TITLE
Clarify new password error message in old v8 branch

### PR DESCRIPTION
**Pull request by jturnbull**
@maxpeterson please merge, this will be released as v.8.1.2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,95 @@
-## v8.1.2 (Backport of v12.0.1)
+## v13.2.0 (Upcoming)
+
+* Clarify error message when your old and new passwords match, you will need to update translations.
```
Comment (meshy) [inline CHANGELOG.md]: If translations need to be updated, this is a backwards incompatible release. `v14`, please

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
-## v8.1.2 (Backport of v12.0.1)
+## v8.1.2
+
+**This release backports specific changes from v12.0.1 and v14.0.0**
```
Comment (meshy) [inline CHANGELOG.md]: `v8.1.2` has already been released with changes from `v12.0.1`. I think that this should be `v8.1.3`

Comment (jturnbull) [inline CHANGELOG.md]: The CHANGELOG in master said 8.1.2 was an upcoming release, but I've checked Pypi, and you're right, it was released.

This is why CHANGELOGs should be the same across branches.

Comment (meshy) [inline CHANGELOG.md]: Yeah, looks like an omission on my part there, sorry.

**Comment by jturnbull** [comment]
Updated again, hopefully that's the end of it

**Comment by jturnbull** [comment]
@mesh, cherry-picked, but still no passes. Any other ideas?

**Comment by meshy** [comment]
not really `:(`

**Comment by Ian-Foote** [comment]
It's a new error. Do we need to backport more travis config?

**Comment by jturnbull** [comment]
Well spotted @Ian-Foote, brought it more from master

**Comment by jturnbull** [comment]
Nope, `ValueError: I/O operation on closed file.`

**Comment by adam-thomas** [comment]
Merged v8.1.x in (containing my now-merged PR https://github.com/incuna/django-user-management/pull/141).

**Comment by meshy** [comment]
HWTI

**Comment by adam-thomas** [comment]
:+1: thanks!

**Pull request merged**